mllib/src/main/scala/org/apache/spark/mllib/clustering/LDA.scala

```diff
@@ -68,6 +52,8 @@ class LDA private (
   def this() = this(k = 10, maxIterations = 20, docConcentration = -1, topicConcentration = -1,
     seed = Utils.random.nextLong(), checkpointInterval = 10)
 
+  private var ldaOptimizer: LDAOptimizer = getDefaultOptimizer("EM")
```

Comment:
I'd list this with the other private vars above and initialize it in the default constructor this()